cloud/src/meta-service/meta_service.cpp

```diff
@@ -1775,7 +1775,7 @@ static bool check_delete_bitmap_lock(MetaServiceCode& code, std::string& msg, st
         code = MetaServiceCode::LOCK_EXPIRED;
         return false;
     }
-    if (lock_id == COMPACTION_DELETE_BITMAP_LOCK_ID) {
+    if (use_new_way && lock_id == COMPACTION_DELETE_BITMAP_LOCK_ID) {
```

Comment:
@zhannngchen  check_delete_bitmap_lock reserve the old way, so i just use this bool use_new_way to decide which branch to enter